src/main/java/gregtech/integration/theoneprobe/provider/MultiRecipeMapInfoProvider.java

```diff
@@ -35,7 +35,7 @@ protected void addProbeInfo(@Nonnull IMultipleRecipeMaps iMultipleRecipeMaps, @N
             if (recipeMap.equals(iMultipleRecipeMaps.getCurrentRecipeMap())) {
                 iProbeInfo.text("   " + TextStyleClass.INFOIMP + "{*recipemap." + recipeMap.getUnlocalizedName() + ".name*} {*<*}");
             } else {
-                iProbeInfo.text("   " + TextStyleClass.LABEL + recipeMap.getLocalizedName());
+                iProbeInfo.text("   " + TextStyleClass.LABEL + "{*recipemap." + recipeMap.getUnlocalizedName() + ".name*}");
```

Comment:
You should use `RecipeMap#getTranslationKey` here instead, so functionality will still work if the lang format changes.